tests/common.js

```diff
@@ -83,7 +83,7 @@ module.exports = {
 
   restartRadar: function(radar, configuration, clients, callback) {
     var tracker = Tracker.create('server restart, given clients ready', function() {
-      if (callback) setTimeout(callback,0);
+      if (callback) setTimeout(callback,5);
     });
 
```

Comment:
```if (callback) setTimeout(callback, 5)```

... include the space before the 5, as you are standardizing on that throughout your changes.